### DISCUSSION TITLE
We should add a the API doc link from the Demo details page.

### DISCUSSION THREAD

**Discussion by atapas** [atapas/webapis-playground#102] [Ideas]
I think we should have an enhancement to add the API doc link from the demo page. Maybe a floating button beside the see code?

![image](https://user-images.githubusercontent.com/3633137/137703858-eb53d105-566a-4b38-afd7-e4ff52294ca7.png)

Should we ask if there are any API doc as part of the PLOP template>

What do you all think?

@mverissimo @wsameer @max-programming @niteshseram @avneesh0612

**Comment by wsameer** [comment]
> I think we should have an enhancement to add the API doc link from the demo page. Maybe a floating button beside the see code?
> 
> ![image](https://user-images.githubusercontent.com/3633137/137703858-eb53d105-566a-4b38-afd7-e4ff52294ca7.png)
> 
> Should we ask if there are any API doc as part of the PLOP template>
> 
> What do you all think?
> 
> @mverissimo @wsameer @max-programming @niteshseram @avneesh0612

That sounds like a good idea. Can we add the link to the title itself? It'd be good to utilize the existing content of the page.

**Reply by atapas** [reply]
@wsameer Could you please provide a bit more details about it? Did you mean a link/icon beside the title?

**Reply by wsameer** [reply]
> @wsameer Could you please provide a bit more details about it? Did you mean a link/icon beside the title?

There's an icon and label already present. As you can see in your attached image "[icon] Network Info" is there. Instead of `<h1/>`, we can make it an `<a/>` tag and use it to link to the document page. This way we can utilize the existing component in a more efficient way.

![image](https://user-images.githubusercontent.com/8468992/137709370-0aab9957-ac9c-4d6c-ae53-465ed93073f7.png)

**Reply by atapas** [reply]
Ok got it. But I am not sure if that link is intuitive enough for someone to recognize as to click on it. I would surely look for other opinions from here. But I like the way you are moving to the solution.

**Reply by mverissimo** [reply]
Maybe a floating button is better, I don't like to remove the `<h1 />` from the page to preserve the semantic and SEO, but we can think of other solutions.

**Reply by atapas** [reply]
Yeah, I would incline to that too.

**Reply by atapas** [reply]
@mverissimo Could you please add one for the API doc? It should read from the demo metadata(we have to add new property now) and show when the property is available? If it is ok for you, I can get you the API link for all existing demos.

**Reply by mverissimo** [reply]
Sure, what is exactly we need to add?

**Reply by atapas** [reply]
Like the see code floating button, we can add another to see API. When someone clicks on the button, it opens a new tab directing to the MDN API page for that particular demo. Does it sound right?

**Reply by mverissimo** [reply]
Yep, looks good! I will develop the new interface thinking on this.

**Comment by atapas** [comment]
Closing this discussion as it is done.
